collection_requirements.rst

```diff
@@ -86,6 +86,16 @@ Example: `meta/runtime.yml <https://github.com/ansible-collections/collection_te
 Modules & Plugins
 ------------------
 
+* Collections MUST only put plugin types recognized by ansible-core into the :file:`plugins/` directory at this time.  The recognized plugin types are listed on https://docs.ansible.com/ansible/devel/plugins/plugins.html plus ``terminal``, ``modules``, and ``module_utils``.
```

Comment:
The GitHub RST renderer does not know `:file:`:
```suggestion
* Collections MUST only put plugin types recognized by ansible-core into the ``plugins/`` directory at this time.  The recognized plugin types are listed on https://docs.ansible.com/ansible/devel/plugins/plugins.html plus ``terminal``, ``modules``, ``doc_fragments`` and ``module_utils``.
```
Maybe we should also link to https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/loader.py and say that if something is supported there, but not included in the docs link and in our explicit list, it is also supported.